screens/ClearEpisodeScreen.tsx

```diff
@@ -0,0 +1,87 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import React, { useState } from "react";
+import {
+  Keyboard,
+  StyleSheet,
+  TextInput,
+  TouchableWithoutFeedback,
+} from "react-native";
+import { Button, Text, View } from "../components/Themed";
+import Colors from "../constants/Colors";
+import { useMainNavigation } from "../hooks/useMainNavigation";
+import { NavigationScreens } from "../navigation/MainNavigationContext";
+import { continueButtonStyle } from "../utils/StylingUtils";
+
+export const ClearEpisodeScreen = () => {
+  const mainNavigation = useMainNavigation();
+  const [password, setPassword] = useState("");
+
+  const validatePassword = async () => {
+    if (password === "aVocado!") {
+      await AsyncStorage.clear();
+      mainNavigation.navigate({ type: NavigationScreens.onboarding });
```

Comment:
This is more of a general comment:

Ok so we clear storage and redirect ourselves to the onboarding screen to reset the study. What I want to ensure is that we don't actually start counting recording days again.

If I reset today and I ship the device out to the participant in two days, we need to ENSURE that when the participant gets the iPad and uses the app it is still day 1.

If we don't already ensure this, the fix would probably be writing the start day of the study into local storage once we make past the initial onboarding screen of the app

Reply:
That is a good point. I am unsure of the exact behavior as of now in regards to the first day.

Reply:
What a ridiculous question, it was ezpz to figure this one out cuz of Iman and my MASSIVE BRAIN